tests/integration/Owncloud/OcisPhpSdk/ShareReceivedTest.php

```diff
@@ -57,11 +57,37 @@ public function testGetAttributesOfReceivedShare(): void
         // $this->assertSame($this->personalDrive->getType(), $receivedShare->getParentDriveType());
         // etags returned by sharedWithMe is not quoted, see https://github.com/owncloud/ocis/issues/8045
         // $this->assertSame($this->fileToShare->getEtag(), $receivedShare->getEtag());
-        $this->assertSame($this->fileToShare->getId(), $receivedShare->getRemoteItemId());
-        $this->assertSame($this->fileToShare->getName(), $receivedShare->getRemoteItemName());
-        $this->assertSame($this->fileToShare->getSize(), $receivedShare->getRemoteItemSize());
-        $this->assertEqualsWithDelta(time(), $receivedShare->getRemoteItemSharedDateTime()->getTimestamp(), 120);
-        $this->assertStringContainsString('Admin', $receivedShare->getOwnerName());
-        $this->assertMatchesRegularExpression('/' . $this->getUUIDv4Regex() . '/', $receivedShare->getOwnerId());
+        $this->assertSame(
+            $this->fileToShare->getId(),
+            $receivedShare->getRemoteItemId(),
+            "Shared Resource Id doesn't match with Resource Id"
+        );
+        $this->assertSame(
+            $this->fileToShare->getName(),
+            $receivedShare->getRemoteItemName(),
+            "Expected Shared Resource Name be ".$this->fileToShare->getName()
+            ." but found ".$receivedShare->getRemoteItemName()
```

Comment:
```suggestion
            "Expected Shared Resource Name be " . $this->fileToShare->getName()
            . " but found " . $receivedShare->getRemoteItemName()
```

Do we put spaces to separate the concat `.` operator? (CI code style is not enforcing consistentcy)